src/pages/dashboard/components/navBar/NavBar.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from '@reach/router';
+import { Trans } from '@lingui/macro';
+
+function NavBar({ configuration, currentPath }) {
+  return (
+    <nav className="fixed pin-b pin-x bg-white md:pin-none md:pin-y md:pin-l ">
+      <ul role="tablist" className="list-reset flex px-0 md:px-3 md:flex-col lg:px-4">
+        {configuration.tabs.map((tabConfig) => {
+          const isSelected = currentPath.indexOf(tabConfig.group) !== -1;
```

Comment:
I'm trying but I don't know how to do the IconActive with that... I don't see any way to show the Icon as active or not using that approach. Check here for last changes: https://github.com/Coding-Coach/coding-coach/pull/307/commits/95a67ce6c20de96f700c558d391a5a00d4149882

On the other hand, we can't do unit testing of this component since we would have to mock the routing and the translations. Any idea then how to test it? @crysfel